doc/sphinx/addendum/parallel-proof-processing.rst

```diff
@@ -154,17 +154,29 @@ to a worker process. The threshold can be configured with
 Batch mode
 ---------------
 
+   .. warning::
+
+      The ``-vio`` flag is subsumed, for most practical usage, by the
+      the more recent ``-vos`` flag. See :ref:`compiled-interfaces`.
```

Comment:
Maybe it could make sense to explain the difference there. Unless it is already done somewhere else.